customer-wallet/src/containers/Transactions/index.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react'
+import { shape, bool, any, object, func, number } from 'prop-types'
+import { Link } from 'react-router-dom'
+// redux
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+// helpers
+import get from 'lodash/get'
+// hoc
+import withCustomerTransactions from '../Hoc/withCustomerTransactions'
+import withFxRates from '../Hoc/withFxRates'
+// components
+import { Card, CardHeader, Avatar, withStyles } from '@material-ui/core';
+import TransactionInfo from './TransactionInfo'
+// icons
+import WalletIcon from '@material-ui/icons/AccountBalanceWallet'
```

Comment:
The use of icons will get interesting once we put in other cryptos. But this is grand for now. Just taking a note really in case that triggers something in terms of organising code in future.